.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - 2.7
   - 3.6
 addons:
-  firefox: 53.0
+  firefox: 54.0
```

Comment:
This used to be `latest` but we had to pin while we were migrating to GeckoDriver/Marionette. Let's switch back to `latest` so we don't have to keep bumping it.